packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerHeader.tsx

```diff
@@ -49,11 +47,50 @@ export const NavigationDrawerHeader = ({
   showCollapseButton,
 }: NavigationDrawerHeaderProps) => {
   const isMobile = useIsMobile();
+  const workspaces = useRecoilValue(workspacesState);
+  const currentWorkspace = useRecoilValue(currentWorkspaceState);
+
+  const { swtitchWorkspace } = useWorkspaceSwitching();
+
+  const handleChange = async (workspaceId: string) => {
+    await swtitchWorkspace(workspaceId);
+  };
 
   return (
     <StyledContainer>
-      <StyledLogo logo={logo} />
-      <StyledName>{name}</StyledName>
+      {!workspaces ? (
+        <Select
+          dropdownId="workspace-selector"
+          fullWidth={true}
+          options={[
+            {
+              label: name,
```

Comment:
maybe rename to something more clear like placeholderName / placeholderLogo if that's what they are? Also maybe (not sure but would probably look better) put the ternary statement inside the Select component options instead of outside it

Reply:
I feel like the naming convention is fine as far as `Select` options go - `label`, `value` is what we get when we define options, isn't it? If it makes sense to change it from a readability standpoint then we can change it! Let me know 😄

Reply:
sorry probably not clear from my comment! I was talking about the values, not the keys (so line 45-46, with the base64 content)

Also my comment on ternary probably isn't relevant anymore since orally we said we might not want it to change style at all if there is only a single option available (no more select component then, just keep current design)

Reply:
Haha, yeah now that makes sense 👍